src/backend/controllers/battleController.js

```diff
@@ -186,16 +186,8 @@ exports.reportResults = async (req: Request, res: Response) => {
 			const newUserOneElo = eloRating.ifTies(userOne.stats.elo, userTwo.stats.elo);
 			const newUserTwoElo = eloRating.ifTies(userTwo.stats.elo, userOne.stats.elo);
 			
-			// We want eloExchanged to be a positive value so we take whoever's increased and subtract 
-			if (userOne.stats.elo > userTwo.stats.elo) {
-				const eloExchanged = userOne.stats.elo - newUserOneElo;
-				battle.eloExchanged = eloExchanged;
-			} else {
-				const eloExchanged = userTwo.stats.elo - newUserTwoElo;
-				battle.eloExchanged = eloExchanged;
-
-			}
-			
+			// Record elo exchanged on tie
+			battle.eloExchanged = Math.abs(userOne.stats.elo-newUserOneElo);			
```

Comment:
Nice

Reply:
thanks for changing this lol

Reply:
Lol no problem. It was really @SecondThread that had the recommendation of how to replace it. I just added it in here since I was updating the logic here anyway.